scripts/get-current-version.sh

```diff
@@ -22,8 +22,8 @@ usage() {
 declare mydir
 mydir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
 
-version_type=${1} # Can be any of these values Build | ReleaseCandidate | Patch | Minor | Major
-current_version=$(cat ${mydir}/../hopr/hopr-lib/Cargo.toml | grep "^version = .*$" | cut -d' ' -f3 | tr -d '"')
+version_type=${1:-semver}
+current_version=$(grep -E '^version\s*=' ${mydir}/../hopr/hopr-lib/Cargo.toml | awk -F\" '{print $2}')
```

Comment:
Instead of this entire file... wouldn't it be easier to install a dependency like this into the base shell: https://github.com/clux/lq

and then do `cat ${mydir}/../hopr/hopr-lib/Cargo.toml | lq '.pacage.version'`?